modules/auxiliary/admin/kerberos/get_ticket.rb

```diff
@@ -190,4 +227,67 @@ def action_get_tgs
     end
   end
 
+  def action_get_hash
```

Comment:
Is this similar enough to how certifried gets the hash that this should be moved in to a shared library?
not suggesting it right now as part of this PR, just wondering if it's planned for later or if there's some reason why that's not a good idea

Reply:
It is similar. I moved and consolidated the U2U request code, but I left the processing for the NTLM extraction in the respective modules. I'm not confident that the NTLM extraction should be in the authenticator, but to your point maybe there's another library I could put it in.

Reply:
It is similar. I moved and consolidated the U2U request code, but I left the processing for the NTLM extraction in the respective modules. I'm not confident that the NTLM extraction should be in the authenticator, but to your point maybe there's another library I could put it in.